mxcube3/ui/actions/general.js

```diff
@@ -165,6 +165,38 @@ export function getInitialState() {
         'Content-type': 'application/json'
       }
     });
+    const loadedSample = fetch('mxcube/api/v0.1/sample_changer/loaded_sample', {
```

Comment:
Only one route for getting the initial state in `actions.general.js`,  we can keep the rest of the routes as they are